dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IEEE14/TestCases/IEEE14DisconnectLine.mo

```diff
@@ -0,0 +1,230 @@
+  Electrical.Controls.Basics.SetPoint QrefPu_Load14(Value0 = Q0Pu_Load14);
+
+  parameter Types.ActivePowerPu P0Pu_Load2 = 0.217000;
+  parameter Types.ReactivePowerPu Q0Pu_Load2 = 0.127000;
+  parameter Types.ActivePowerPu P0Pu_Load3 = 0.942000;
+  parameter Types.ReactivePowerPu Q0Pu_Load3 = 0.190000;
+  parameter Types.ActivePowerPu P0Pu_Load4 = 0.478000;
+  parameter Types.ReactivePowerPu Q0Pu_Load4 = -0.039000;
+  parameter Types.ActivePowerPu P0Pu_Load5 = 0.076000;
+  parameter Types.ReactivePowerPu Q0Pu_Load5 = 0.016000;
+  parameter Types.ActivePowerPu P0Pu_Load6 = 0.112000;
+  parameter Types.ReactivePowerPu Q0Pu_Load6 = 0.075000;
+  parameter Types.ActivePowerPu P0Pu_Load9 = 0.295000;
+  parameter Types.ReactivePowerPu Q0Pu_Load9 = 0.166000;
+  parameter Types.ActivePowerPu P0Pu_Load10 = 0.090000;
+  parameter Types.ReactivePowerPu Q0Pu_Load10 = 0.058000;
+  parameter Types.ActivePowerPu P0Pu_Load11 = 0.035000;
+  parameter Types.ReactivePowerPu Q0Pu_Load11 = 0.018000;
+  parameter Types.ActivePowerPu P0Pu_Load12 = 0.061000;
+  parameter Types.ReactivePowerPu Q0Pu_Load12 = 0.016000;
+  parameter Types.ActivePowerPu P0Pu_Load13 = 0.135000;
+  parameter Types.ReactivePowerPu Q0Pu_Load13 = 0.058000;
+  parameter Types.ActivePowerPu P0Pu_Load14 = 0.149000;
+  parameter Types.ReactivePowerPu Q0Pu_Load14 = 0.050000;
+
+equation
+  // Loads references
+  Load2.PRefPu = PrefPu_Load2.setPoint.value;
+  Load2.QRefPu = QrefPu_Load2.setPoint.value;
+  Load3.PRefPu = PrefPu_Load3.setPoint.value;
+  Load3.QRefPu = QrefPu_Load3.setPoint.value;
+  Load4.PRefPu = PrefPu_Load4.setPoint.value;
+  Load4.QRefPu = QrefPu_Load4.setPoint.value;
+  Load5.PRefPu = PrefPu_Load5.setPoint.value;
+  Load5.QRefPu = QrefPu_Load5.setPoint.value;
+  Load6.PRefPu = PrefPu_Load6.setPoint.value;
+  Load6.QRefPu = QrefPu_Load6.setPoint.value;
+  Load9.PRefPu = PrefPu_Load9.setPoint.value;
+  Load9.QRefPu = QrefPu_Load9.setPoint.value;
+  Load10.PRefPu = PrefPu_Load10.setPoint.value;
+  Load10.QRefPu = QrefPu_Load10.setPoint.value;
+  Load11.PRefPu = PrefPu_Load11.setPoint.value;
+  Load11.QRefPu = QrefPu_Load11.setPoint.value;
+  Load12.PRefPu = PrefPu_Load12.setPoint.value;
+  Load12.QRefPu = QrefPu_Load12.setPoint.value;
+  Load13.PRefPu = PrefPu_Load13.setPoint.value;
+  Load13.QRefPu = QrefPu_Load13.setPoint.value;
+  Load14.PRefPu = PrefPu_Load14.setPoint.value;
+  Load14.QRefPu = QrefPu_Load14.setPoint.value;
+  Load2.deltaP = 0;
+  Load2.deltaQ = 0;
+  Load3.deltaP = 0;
+  Load3.deltaQ = 0;
+  Load4.deltaP = 0;
+  Load4.deltaQ = 0;
+  Load5.deltaP = 0;
+  Load5.deltaQ = 0;
+  Load6.deltaP = 0;
+  Load6.deltaQ = 0;
+  Load9.deltaP = 0;
+  Load9.deltaQ = 0;
+  Load10.deltaP = 0;
+  Load10.deltaQ = 0;
+  Load11.deltaP = 0;
+  Load11.deltaQ = 0;
+  Load12.deltaP = 0;
+  Load12.deltaQ = 0;
+  Load13.deltaP = 0;
+  Load13.deltaQ = 0;
+  Load14.deltaP = 0;
+  Load14.deltaQ = 0;
+
+  // Generators controls and references
+  ModelSignalN.thetaRef = Theta_Bus1;
```

Comment:
The following 7 equations (+ the associated variables and blocks) could be in the BaseClasses